Mathlib/Order/Interval/Finset/Nat.lean

```diff
@@ -236,6 +236,22 @@ theorem multiset_Ico_map_mod (n a : ℕ) :
 
 end Multiset
 
+section List
+
+namespace List
+
+lemma toFinset_range'_1 (a b : ℕ) : (List.range' a b).toFinset = Ico a (a+b) := by
```

Comment:
What's the `1` here?

Reply:
It's the third, autoparam argument in `List.range'`, and this naming style is consistent with Lean 4 naming, eg [`List.mem_range'_1`: ](https://loogle.lean-lang.org/?q=%22range%27_1%22%2C+List)